Editor/EditorCore/TooltipEditor.cs

```diff
@@ -12,6 +13,22 @@ namespace UnityEditor.ProBuilder
     /// </summary>
     sealed class TooltipEditor : EditorWindow
     {
+#if !UNITY_2019_1_OR_NEWER
+        static TooltipEditor()
+        {
+            s_ShowModeEnum = ReflectionUtility.GetType("UnityEditor.ShowMode");
+            Assert.IsNotNull(s_ShowModeEnum, "Failed to get ShowMode enum");
```

Comment:
In other places where reflection is used, we don't `Assert` or throw errors unless in Debug mode.

I think in this case we should first try to use the reflected version, but on failure, fall back to the publicly exposed `ShowPopup`.